src/Kernel/client/kernel.ts

```diff
@@ -38,29 +85,41 @@ class Kernel {
                     if (div != null) {
                         let amplitudeSquaredButton = document.createElement("button");
                         let graph = document.createElement("canvas");
+                        graph.id = "first";
```

Comment:
When two `%debug` commands are called in the same notebook, this could wind up giving the same ID to multiple HTML elements, breaking an invariant used by the `document.getElementById` DOM method. In this case, you have a workaround in that you've already got `graph` as a local variable, such that you can use it directly at line 93 instead of looking the element up in the DOM again.